storage/storage_image.go

```diff
@@ -416,8 +416,23 @@ func (s *storageImageDestination) Commit() error {
 	return nil
 }
 
+var manifestMIMETypes = []string{
+	// TODO(runcom): we'll add OCI as part of another PR here
+	manifest.DockerV2Schema2MediaType,
+	manifest.DockerV2Schema1SignedMediaType,
+	manifest.DockerV2Schema1MediaType,
+}
```

Comment:
The PR name says “support manifest list”, but this
- actually _restricts_ MIME types (returning `nil` from `SupportedManifestMIMETypes` meant “anything is acceptable”)
- does not mention manifest lists at all?!

Is the PR really doing what the title says, somehow, or is the title wrong?